packages/css/src/components/blockquote/blockquote.scss

```diff
@@ -34,6 +34,6 @@
   }
 }
 
-.ams-blockquote--inverse-color {
-  color: var(--ams-blockquote-inverse-color);
+.ams-blockquote--color-inverse {
```

Comment:
We don’t include prop names in our classes. This should be `.ams-blockquote-inverse`. And likewise for other components.

Compare: `.ams-button-primary` (variant), `.ams-paragraph-large` (size) etc.

(Let’s discuss and document the rules for this in our next dev refinment.)